tests/Doctrine/Tests/ORM/Decorator/EntityManagerDecoratorTest.php

```diff
@@ -1,112 +0,0 @@
-<?php
-
-declare(strict_types=1);
-
-namespace Doctrine\Tests\ORM\Decorator;
-
-use Doctrine\ORM\Decorator\EntityManagerDecorator;
-use Doctrine\ORM\EntityManagerInterface;
-use Doctrine\ORM\Query\ResultSetMapping;
-use PHPUnit\Framework\MockObject\MockObject;
-use PHPUnit\Framework\TestCase;
-use ReflectionClass;
-use ReflectionMethod;
-
-use function array_fill;
-use function in_array;
-
-class EntityManagerDecoratorTest extends TestCase
```

Comment:
As soon as we add return types, this test is broken beyond repair because it violates the type contracts almost everywhere. 🙁